tests/test_analysis.py

```diff
@@ -238,6 +238,18 @@ def test_simulate_ideal_trajectories(self):
         filename = 'simulate_ideal_trajectories.png'
         savefig(docs_dir, filename)
 
+    def test_get_centered_endpoints(self):
+        traj1 = np.array([[0, 0, 0], [1, 1, 1], [2, 2, 2]])  # Ends at (2,2,2)
+        traj2 = np.array([[0, 0, 0], [1, 0, 1], [3, 1, 3]])  # Ends at (3,1,3)
+        traj3 = np.array([[0, 0, 0], [0, 1, 1], [1, 2, 3]])  # Ends at (1,2,3)
```

Comment:
you could add a trajectory that doesn't begin at the origin to demonstrate the "centering" property